### DISCUSSION TITLE
How to get scrollable element with OverlayScrollbarsComponent (react)?

### DISCUSSION THREAD

**Discussion by krutoo** [KingSora/OverlayScrollbars#476] [Q&A] [answered]
Hi, thank for really helpful library.

I use `OverlayScrollbarsComponent` from react package (v1) 

I need to get element that actually scrolls. Its need to use it with for example `body-scroll-lock` like libraries

How can i get it?

**Comment by KingSora** [comment, marked as answer]
Good day @krutoo 
So for `overlayscrollbars-react` `v0.3.0` and below (the ones compatible with `v1`) you have to do this:
1. Get the `ref` to the component
2. Then  call `ref.current.osInstance().getElements().viewport`

This will give you the `viewport` element which is responsible for scrolling.

I highly recommend to update to `v2` though.. (there the process is similar, just call `elements()` instead of `getElements()`)

**Reply by krutoo** [reply]
Thank you for fastest answer
